src/Data/ProgrammesDb/Entity/Promotion.php

```diff
@@ -119,6 +119,15 @@ class Promotion
     private $cascadesToDescendants = false;
 
     /**
+     * Used in queries only. No getter or setter as creation should be handled
+     * by the RelatedLink entity.
+     *
+     * @ORM\OneToMany(targetEntity="RelatedLink", mappedBy="relatedToPromotion")
+     * @ORM\OrderBy({"position" = "ASC"})
```

Comment:
Do we need to specify that here? It could just be a secondary sort on the query, this seems overkill and I don't trust doctrine not to insert random orderby's in places I don't want them.

Reply:
Good point, turns out you can add that to the query rather than being a implicit all-the-time thing.